src/cpu/arm/fuchsia.rs

```diff
@@ -12,7 +12,7 @@
 // OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
 // CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-use super::{AES, ARMCAP_STATIC, NEON, PMULL, SHA256};
```

Comment:
@erickt We don't have Fuchsia in our CI yet so I'm not sure if these changes are enough to avoid breaking the build. PTAL if you can.

Reply:
Thanks for the ping! I'll test it out and get back to you.

Reply:
PR #2336 is the next step in this which has similar hazards of missing `use`s and whatnot.

Reply:
OK, thanks!